### ISSUE TITLE
bert4torch中，使用单卡多GPU训练，使用accelerate后，不能使用梯度剪裁。

### ISSUE THREAD

**Issue by wjj97172** [Tongjilibo/bert4torch#162]
定义模型的时候，如果使用了accelerate模块后，model.compile()里面不能使用梯度剪裁，

model = AccelerateTrainer(net,**var(args))

model.compile(
loss = nn.CrossEntropyLoss(),
optimizer=optimizer,
clip_grad_norm=1.0,
metrics=['accuracy']  )

报错：accelerateTrainer object has no attribute parameters

**Comment by Tongjilibo** [comment]
可以安装bert4torch==0.4.1使用

**Comment by wjj97172** [comment]
> 可以安装bert4torch==0.4.1使用

升级后，可以使用了。
